.github/workflows/deploy-preview.yml

```diff
@@ -14,7 +14,9 @@
 
 name: Deploy Preview
 
-on: pull_request
+on:
+  pull_request:
+    types: [closed, synchronize, reopened, opened]
```

Comment:
The normal pull_request trigger already contains synchronize, reopened, and opened. So it works already!